slacktee.sh

```diff
@@ -33,35 +33,50 @@ cond_prefix_prefixes=()
 cond_prefix_patterns=()
 found_title_prefix=""
 
+function err_exit() 
+{
+	exit_code=$1
+	shift
+	echo "$@" > /dev/null >&2
+	exit $exit_code
+}
+function cleanup() 
+{
+	rm -rv $tmp_dir
```

Comment:
This is dangerous. This code removes all files in the $tmp_dir.
Since `/tmp` is the default dir of slacktee, this cleans up the temp files created by other applications.
If we have this function, we should add a prefix such as 'slacktee-' to our temp file.